---
title: "Identification of RNA pseudouridine sites using deep learning approaches"
authors:
- admin
- Md. Al Mehedi Hasan
- Jungpil Shin
# author_notes:
# - "Equal contribution"
# - "Equal contribution"
date: "2021-02-23T00:00:00Z"
doi: "https://doi.org/10.1371/journal.pone.0247511"

# Schedule page publish date (NOT publication's date).
publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Plos one 16.2 (2021): e0247511."
publication_short: ""

abstract: Pseudouridine(Ψ) is widely popular among various RNA modifications which have been confirmed to occur in rRNA, mRNA, tRNA, and nuclear/nucleolar RNA. Hence, identifying them has vital significance in academic research, drug development and gene therapies. Several laboratory techniques for Ψ identification have been introduced over the years. Although these techniques produce satisfactory results, they are costly, time-consuming and requires skilled experience. As the lengths of RNA sequences are getting longer day by day, an efficient method for identifying pseudouridine sites using computational approaches is very important. In this paper, we proposed a multi-channel convolution neural network using binary encoding. We employed k-fold cross-validation and grid search to tune the hyperparameters. We evaluated its performance in the independent datasets and found promising results. The results proved that our method can be used to identify pseudouridine sites for associated purposes.

# Summary. An optional shortened abstract.
summary: In this paper, we proposed a multi-channel convolution neural network using binary encoding for identifying Pseudouridine(Ψ) sites from nucleotide sequences.

tags:
- Bioinformatics, Deep Learning
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0247511&type=printable
url_code: 
url_dataset: https://doi.org/10.1371/journal.pone.0247511.s001
url_poster: ''
url_project: ''
url_slides: https://docs.google.com/presentation/d/1PrRGSFcO8OM8XBOhGp52K2O8_cN9J3R5/edit?usp=sharing&ouid=112432914184650588612&rtpof=true&sd=true
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

{{% callout note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /callout %}}

{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}

Supplementary notes can be added here, including [code, math, and images](https://wowchemy.com/docs/writing-markdown-latex/).
